package.json

```diff
@@ -22,7 +22,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "CI=false && react-scripts build",
```

Comment:
You are not supposed to change the build script.
Return it back `react-scripts build`.